ls1intum/Artemis
# Chore / Fix issues with LanguageHelper in tests

### Checklist

### Motivation and Context



Fixes issues with the instantiation of the LanguageHelper in jest tests.

### Description


The problem was that the mock of the LanguageHelper did not have the language getter.
Because of this tests that are declared with a module that uses the LanguageHelper offen had the issue that `this.languageHelper.language` is undefined.
This issue could be circumvented by changing the import of the LanguageHelper in the modules, I also corrected these imports.

### Steps for Testing


1. Log in to Artemis
2. Visit an area that uses the LanguageHelper (e.g. Course Administration)
3. Change the language => The translations in the UI should change

### Screenshots
